sources/vega/data/v1/spec.proto

```diff
@@ -1,25 +1,53 @@
 syntax = "proto3";
 
-package oracles.v1;
+package data.v1;
 
-option go_package = "code.vegaprotocol.io/protos/vega/oracles/v1";
+option go_package = "code.vegaprotocol.io/protos/vega/data/v1";
 
-// An oracle spec describe the oracle data that a product (or a risk model)
-// wants to get from the oracle engine.
-message OracleSpecConfiguration {
+message DataSourceDefinition {
+  oneof type {
+    InternalDataSourceDefinition internal = 1;
+    ExternalDataSourceDefinition external = 2;
+  }
+}
+
+message InternalDataSourceDefinition {
+  // Types of internal data sources
+  oneof type {
+    TimeDataSourceDefinition time = 1;
+  }
+}
+
+// TimeDataSource is the internal data source used for emitting timestamps.
+message TimeDataSourceDefinition {
+  // Conditions that the timestamps should meet in order to be considered.
+  repeated Condition conditions = 1;
+}
+
+message ExternalDataSourceDefinition {
+  // Types of External data sources
+  oneof type {
+    DataSpecConfiguration oracle = 1;
+    DataSpecConfiguration bridge = 2;
```

Comment:
Not sure I get what these 2 are?

Reply:
This is a oneof setting for different type of external spec configuration - we can define an oracle or a bridge.
But this was added in order to show how different external types would be treated anyway, so I'll just leave oracle for now.

Reply:
Removed the unused one